app/views/reservations/_reservation_fields.html.haml

```diff
@@ -1,10 +1,12 @@
 .datetime-block
   - start_disabled = start_time_disabled?(f.object)
-  = f.input :reserve_start_date, :input_html => { :class => 'datepicker', disabled: start_disabled }
+  - date_field = f.object.actual_start_at ? :actual_start_date : :reserve_start_date
+  = f.input date_field, :input_html => { :class => 'datepicker', disabled: start_disabled }
```

Comment:
I think you might already be checking this in the controller, but make sure I can't hack the HTML to get it enabled, submit an actual_start_date, and have it persist.